packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,32 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement made by the Flutter framework.
+///
+/// This class is mostly used by the testing api to store the announcements
+/// in a user-friendly structure so that the code consumers can check the details
+/// of announcements in their tests.
+///
+/// See [TestWidgetsFlutterBinding.takeAnnouncements].
+class CapturedAccessibilityAnnouncement {
+
+  /// Creates a [CapturedAccessibilityAnnouncement].
```

Comment:
That this constructor creates a `CapturedAccessibilityAnnouncement` is obvious. Consider writing an answer a user may have when trying to use this constructor (https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#answer-your-own-questions-straight-away). Alternatively, make it private, since tests are not meant to invoke it directly. Then you won't need to document it as it's trivial.

See also https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation

Reply:
Thanks for the resources :)
I Made the constructor private.